ui/summary.go

```diff
@@ -277,7 +286,13 @@ func SummarizeMetrics(w io.Writer, indent string, t time.Duration, timeUnit stri
 		if sink, ok := m.Sink.(*stats.TrendSink); ok {
 			cols := make([]string, len(TrendColumns))
 			for i, col := range TrendColumns {
-				value := m.HumanizeValue(col.Get(sink), timeUnit)
+				var value string
+				if col.Key == "count" {
```

Comment:
I don't like the several `stat == "count"` checks in this file, nor the unnecessary conversion from `uint64` to `float64` to `string` for `trendCountColumn`, but it was the most straightforward way to add this without refactoring the core logic in `SummarizeMetrics()`. Let me know if there's a cleaner way (i.e. less code and conditionals) to make this happen.

Reply:
@imiric If you worry about needless code, then you can always do `v := col.Get(sink)` then use v instead of calling `col.Get(sink)` twice. In Go at least, the `else` part is discourage because it often makes the code less readable.

Reply:
@cuonglm I think you meant this as a response to the other thread, but even with isolating `col.Get(sink)` you would need an `else` branch to pass it to `m.HumanizeValue()`.

I agree with the Go convention of dropping `else`, it is more readable that way, but I'd rather leave it for cases when it doesn't execute code for no reason other than because it's more readable.